docs/dev/existing_data_updates.rst

```diff
@@ -343,6 +334,28 @@ above. Continue to iterate and debug until assets generate successfully.
 create new tables in :mod:`pudl.metadata.resources.nrelatb` for these descriptors,
 following the example of ``core_nrelatb__yearly_technology_status``.
 
+E. FERC Form 714
+^^^^^^^^^^^^^^^^
+**4.E.1)** Materialize everything downstream of the raw FERC-714 assets using Dagster
+query ``key:"raw_ferc714_xbrl*"+``. Investigate any errors that occur, and update the
+constants in :mod:`pudl.transform.ferc714` to add any new fix cases for the new year
+of data. Common updates include:
+
+* :py:const:`pudl.transform.ferc714.TIMEZONE_OFFSET_CODE_FIXES` - Update this if you
+  see ``AssertionError: We expect all but XX of the records without a cleaned
+  utc_offset to not have any demand data, but we found YY records``, after
+  investigating the records with missing utc_offset and determining what the correct
+  value should be.
+* :py:const:`pudl.transform.ferc714.DISCONTINUOUS_DATES` - Update this if you see
+  ``AssertionError: We expect there to be fewer than XX gaps in the xbrl time series
+  but we found these YY gaps:``, after investigating the new gaps and confirming
+  they occur on reasonable dates (usually around daylight saving time start or end).
+* :py:const:`pudl.transform.ferc714.DUPLICATED_DATETIMES` - Update this if you see
+  ``AssertionError: Found YY duplicate UTC datetimes, but we expected XX or less``,
+  after investigating the new duplicates and confirming they occur on reasonable
+  dates (usually when a respondent changes timezones, whether due to daylight saving
+  time or otherwise).
```

Comment:
Editing just because imprecise definitions of "timezone" cause so much data suffering.

```suggestion
* :py:const:`pudl.transform.ferc714.DUPLICATED_DATETIMES` - Update this if you see
  ``AssertionError: Found YY duplicate UTC datetimes, but we expected XX or less``,
  after investigating the new duplicates and confirming they occur on reasonable
  dates (usually when a respondent changes their UTC offset, whether due to daylight
  savings time or otherwise).
```